irods_http_client/user_group_operations.py

```diff
@@ -0,0 +1,809 @@
+                print('Failed to add user \'' + user + '\' to group \'' + group +  '\' : iRODS Status Code' + str(rdict['irods_response']['status_code']))
+            else:
+                print('User \'' + user + '\' added to group \'' + group +  '\' successfully')
+            
+            return(
+                {
+                    'status_code': r.status_code,
+                    'data': rdict
+                }
+            )
+        else:
+            irods_err = ''
+            rdict = None
+            if (r.text != ''):
+                rdict = r.json()
+                irods_err = ': iRODS Status Code' + str(rdict['irods_response'])
+            print(f'Error <{r.status_code}>{irods_err}')
+
+            return(
+                {
+                    'status_code': r.status_code,
+                    'data': rdict
+                }
+            )
+
+
+    def remove_from_group(self, user: str, zone: str, group: str):
+        """
+        Removes a user from a group. Requires rodsadmin or groupadmin privileges.
+        
+        Parameters
+        - user: The user to be removed from the group.
+        - zone: The zone for the user to be removed from the group.
+        - group: The group for the user to be removed from.
+
+        Returns
+        - A dict containing the HTTP status code and iRODS response.
+        - The iRODS response is only valid if no error occurred during HTTP communication.
+        """
+        if (self.token == None):
+            raise RuntimeError('No token set. Use setToken() to set the auth token to be used')
+        if (not isinstance(user, str)):
+            raise TypeError('user must be a string')
+        if (not isinstance(zone, str)):
+            raise TypeError('zone must be a string')
+        if (not isinstance(group, str)):
+            raise TypeError('group must be a string')
+
+        headers = {
+            'Authorization': 'Bearer ' + self.token,
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+
+        data = {
+            'op': 'remove_from_group',
+            'user': user,
+            'zone': zone,
+            'group': group
+        }
+
+        r = requests.post(self.url_base + '/users-groups', headers=headers, data=data)
+
+        if (r.status_code / 100 == 2):
+            rdict = r.json()
+            if rdict['irods_response']['status_code']:
+                print('Failed to remove user \'' + user + '\' from group \'' + group +  '\' : iRODS Status Code' + str(rdict['irods_response']['status_code']))
+            else:
+                print('User \'' + user + '\' removed from group \'' + group +  '\' successfully')
+            
+            return(
+                {
+                    'status_code': r.status_code,
+                    'data': rdict
+                }
+            )
+        else:
+            irods_err = ''
+            rdict = None
+            if (r.text != ''):
+                rdict = r.json()
```

Comment:
If an HTTP error occurs, then the information in `r.text` and `r.json()` can be ignored.

Open an issue for removing these lines throughout the implementation. Link the issue to this conversation. We'll handle this in a separate PR.